### PR TITLE
Toggle Button — Fade variant

### PR DESCRIPTION
**Pull request by vladmoroz**
Added `variant="fade"` to the `ToggleButton`

### CODE REVIEW

File: packages/website/src/docs/toggle-button.mdx [hunk 1]
```diff
@@ -39,6 +39,11 @@ description: Useful for allowing users to select one or more options from a set.
...
       description: 'The content to render',
     },
+    variant: {
+      type: 'normal | fade',
+      default: 'normal',
+      description: 'The variant to apply',
+    },
     size: {
       type: '0 | 1',
       default: '0',
```
Comment (peduarte) [inline packages/website/src/docs/toggle-button.mdx]: unrelated but shouldn't the default be `1`  here?

Comment (vladmoroz) [reply packages/website/src/docs/toggle-button.mdx]: It should! df1ff999877ffb094ff7c28314e8d24c435471eb

File: packages/radix/src/components/ToggleButton.tsx [hunk 2]
```diff
@@ -71,27 +102,23 @@ export const ToggleButton = withTheme(
                  * remove the borders and use box-shadows, because we use negative margins to compensate for adjacent
                  * border.
                  */
-                borderColor: 'transparent',
                 '&:hover': {
-                  borderColor: 'transparent',
+                  borderColor: props.size === 0 ? 'transparent' : 'gray500',
```
Comment (peduarte) [inline packages/radix/src/components/ToggleButton.tsx]: These type of rules where one variant depends on another is worrying me :) 

It's ok when we're hand writing the code like now, but... you know what I mean.

**Comment by peduarte** [comment]
Awesome! LGTM

**Pull request merged**